docs/new_uberon_import.md

```diff
@@ -0,0 +1,15 @@
+# Importing a new UBERON term to CL
+
+Often an UBERON term that is not yet imported by CL will be necessary (e.g. for a nice logic definition). 
+
+For updating the imports, the step-by-step is:
+
+- Make sure you are working on a new branch, in the context of the ticket you are working on 
+
+- Add the UBERON term to the end of the list in `cell-ontology/src/ontology/imports/uberon_terms.txt`
+
+- Run `./run.sh make imports/uberon_import.owl`
```

Comment:
./run.sh requires people to have done a chmod +x beforehand which many have not; maybe better sh run.sh.